src/sass/index.sass

```diff
@@ -283,9 +292,18 @@
               .filter-item
                 padding-top: 1rem
 
-        .sheet-body
-          overflow-y: auto
-          border-top: 1px solid #69675d
+      .sheet-body
+        overflow-y: auto !important
+        border-top: 1px solid #69675d
+
+        .editor
+          height: auto !important
+
+        .editor-edit
+          display: none !important
```

Comment:
For windows with spell/item descriptions the edit button still appeared when clicking the text